sklearn/neighbors/tests/test_lof.py

```diff
@@ -44,14 +47,15 @@ def test_lof():
 
     # Assert predict() works:
     clf = neighbors.LocalOutlierFactor(contamination=0.25, n_neighbors=5).fit(X)
-    assert_array_equal(clf._predict(), 6 * [1] + 2 * [-1])
-    assert_array_equal(clf.fit_predict(X), 6 * [1] + 2 * [-1])
+    expected_predictions = np.asarray(6 * [1] + 2 * [-1], dtype=global_dtype)
```

Comment:
Do you need that ? assert_allclose should do that by itself iirc

Reply:
This was to remove duplication.

Reply:
What I meant is
```suggestion
    expected_predictions = 6 * [1] + 2 * [-1]
```